lib/models/leaderboard_user.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:coffeecard/generated/api/coffeecard_api.swagger.swagger.dart';
+import 'package:equatable/equatable.dart';
+
+class LeaderboardUser extends Equatable {
+  final String name;
+  final int rank;
```

Comment:
This int is for the number of cups drunk, right? So `rank` might be a bit misleading